File: PRAK605-2210817210023-Muhammad Shofy Akmal.py
m = int(input())
print("Matriks A")
m1 = []; m2 = []
for a in range(0, m) :
    isi1 = list(map(int, input().split()))
    m1.append(isi1[0:m])
print("Matriks B")
for a in range(0, m) :
    isi2 = list(map(int, input().split()))
    m2.append(isi2[0:m])
print("\nMatriks A x B")
for a in range(0, m) :
    for b in range(0, m) :
        hasil = 0
        for c in range(0, m) :
            hasil+=m1[a][c]*m2[c][b]
        print(hasil, end=' ')
    print()

File: PRAK605-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
int main () {
int m, a, b, c, isi, h;
scanf ("%d", &m);
int m1[m][m], m2[m][m];
printf ("Matriks A\n");
for (a=0; a<m; a++) {
    for (b=0; b<m; b++) {
        scanf ("%d", &isi);
        m1[a][b]=isi;}}
printf ("Matriks B\n");
for (a=0; a<m; a++) {
    for (b=0; b<m; b++) {
        scanf ("%d", &isi);
        m2[a][b]=isi;}}
printf ("\nMatriks A x B\n");
for (a=0; a<m; a++) {
    for (b=0 ; b<m; b++) {
        for (c=0, h=0; c<m; c++) {
            h+=m1[a][c]*m2[c][b];}
        printf ("%d ", h);}
    printf ("\n");}
}